framework/interface/api_handlers.py

```diff
@@ -842,3 +843,71 @@ def delete(self, error_id=None):
             self.get_component("db_error").Delete(error_id)
         except exceptions.InvalidErrorReference:
             raise tornado.web.HTTPError(400)
+
+class ReportExportHandler(custom_handlers.APIRequestHandler):
+    """
+    Class handling APIs related to export report funtionality.
+    In OWTF, export report funtionality is implemented using docxtemplater.
+    docxtemplater takes two inputs:
+     - input.docx - Docx template for docxtemplater
+     - JSON - Data need to be filled in above template
+    """
+    SUPPORTED_METHODS = ['GET']
+
+    def get(self, target_id=None):
+        """
+        REST API - /api/targets/<target_id>/export/ returns JSON(data) for template.
+        """
+        if not target_id:
+            raise tornado.web.HTTPError(400)
+        try:
+            filter_data = dict(self.request.arguments)
+            plugin_outputs = self.get_component("plugin_output").GetAll(filter_data, target_id=target_id, inc_output=True)
+            # Group the plugin outputs to make it easier in template
+            grouped_plugin_outputs = {}
+            for poutput in plugin_outputs:
+                if grouped_plugin_outputs.get(poutput['plugin_code']) is None:
+                    # No problem of overwriting
```

Comment:
I am not sure to understand the the point of this comment. What is overridden? Maybe I am not understanding the code snippet but I believe that the line below this comment does not override anything.